src/java/edu/ucsd/library/dams/api/DAMSAPIServlet.java

```diff
@@ -5414,10 +5414,10 @@ public static Properties loadConfig() throws Exception
 			
 			addProperty(props, "ffmpeg", varNamePrefix);
 			addProperty(props, "magick.convert", varNamePrefix);
-			addProperty(props, "jhove.conf", varNamePrefix);
-			addProperty(props, "jhove.maxSize", varNamePrefix);
-			addProperty(props, "jhove.zipModel.command", varNamePrefix);
-			addProperty(props, "jhove.gzipModel.command", varNamePrefix);
+
+			// FITS command and configration for file characterization
```

Comment:
Will there be a corresponding PR to private config to populate these property values?

Reply:
@mcritchlow It seems like I am not able to push a new branch to private config:
```
$ git push origin feature/characterization-fits
ERROR: Repository not found.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
Could you check my access rights? Thanks.

Reply:
How odd. Essentially none of us had access.. Must have been a migration issue. Could you try again?

Reply:
Now it works: https://github.com/ucsdlib/private_config/pull/1. Thanks @mcritchlow .